### ISSUE TITLE
[Bug]: VoiceOver accessibility nearly non-existent

### ISSUE THREAD

**Issue by dariaradford95-tech** [simplex-chat/simplex-chat#7051]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Platform

iOS

### OS version

26.6 beta one

### App version

Multiple versions, including current

### Current Behavior

Many screens contain multiple unlabelled buttons or unclearly labelled buttons accessibility is so poor with VoiceOver that I am unable to use this app with a friend who works overseas and refers this app for most communication. For an app that is built for privacy first it should really have accessibility in its make up as well. Screen reader users deserve proper protection too.

### Expected Behavior

All buttons and played elements should be clearly labelled and accessible with VoiceOver and should behave as expected according to those labels I should not be forced to close the app in order to get out of a screen simply because the close button vanishes out of nowhere I should be able to access all information within the application without any difficulty A.

### Steps To Reproduce

Go to the application turn on VoiceOver and begin browsing through every single screen in the application from chat to accounts to any other feature available in the app. The only screen I found completely accessible was The get started screen as well as the profile naming screen.. Everything else is either unclearly labelled or does not behave appropriately with VoiceOver.

### Relevant log output

```shell

```
